wca-regulations.md

```diff
@@ -179,17 +179,17 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 5a) Examples of puzzle defects include: popped parts, pieces twisted in place, and detached screws/caps/stickers.
 - 5b) If a puzzle defect occurs during an attempt, the competitor may choose to either repair the defect and continue the attempt, or to stop the attempt.
     - 5b1) If a competitor chooses to repair the puzzle, they must repair only the defective parts. Tools and/or parts of other puzzles must not be used to repair the original puzzle. Penalty: disqualification of the attempt (DNF).
-    - 5b2) Any repair to a puzzle must not give the competitor any advantage in solving the puzzle. Penalty: disqualification of the attempt (DNF).
+    - 5b2) Any repair to a puzzle must not give the competitor an advantage in solving the puzzle. Penalty: disqualification of the attempt (DNF).
```

Comment:
Isn't "any" also correct?

Reply:
Hmm.. I suppose this could probably go both ways. I think using the word "an" sounds slightly better, but maybe we should get a second opinion.